depenguinme.sh

```diff
@@ -254,33 +254,35 @@ _get_all_valid_drives() {
 	# Capture the complete output of lsblk for debugging
 	lsblk_output=$(lsblk -no NAME,TYPE,TRAN)
 
-        # Count the number of NVMe drives (doesn't work with hetzner)
-        nvme_count=$(echo "$lsblk_output" | grep -c '^nvme')
+	# Count the number of NVMe drives (doesn't work with hetzner)
```

Comment:
What does this comment mean? What exactly doesn't work with Hetzner?

Reply:
I think this is a carry over from earlier version when we used another tool on Hetzner hosts for nvme drives, and I didn't update the comment